portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/UpgradeReport.testcase

```diff
@@ -349,7 +349,7 @@ definition {
 	@priority = 4
 	test ReportGeneratedWithLogContext {
 		property custom.properties = "upgrade.report.enabled=true${line.separator}upgrade.log.context.enabled=true";
-		property database.partition.enabled = "true";
+		property database.auto.upgrade.enabled = "true";
 		property log.context.enabled = "true";
 		property skip.start.app.server = "false";
 		property skip.upgrade-legacy-database = "true";
```

Comment:
I think we are missing fix `UserNotifiedWhereToFindUpgradeReportAfterAutoUpgrade`, that was also changed to db partition property instead of auto.upgrade